components/authz-service/server/v2/project_update_manager.go

```diff
@@ -269,6 +288,30 @@ func (manager *ProjectUpdateManager) startProjectUpdateForDomainResources(
 	return err
 }
 
+func (manager *ProjectUpdateManager) cancelProjectUpdateForDomainResources() error {
```

Comment:
The only thing needed to cancel a project update for the domain services is to publish a 'cancel' event.